R/lt_model_lq.R

```diff
@@ -56,6 +54,7 @@
 #'  \item{values}{ Associated values of \code{q0_5, q0_1, q15_45, q15_35} 
 #' and \code{e0}.}
 #' @seealso \code{\link[MortalityEstimate]{wilmothLT}}
```

Comment:
We can link to packages on github if they're listed in `Remotes: mpascariu/MortalityEstimate` in the` DESCRIPTION` file (I think), but would need to check if it'd also need to be added to Suggests: or similar. Also need to check if a dependency is required or not to use `Remotes:` in the first place. You can see this in use in the [DDSQLtools](https://github.com/timriffe/DDSQLtools/blob/master/DESCRIPTION) package, which depends on this one.